src/bootstrap/bootstrap.py

```diff
@@ -458,23 +458,52 @@ def unpack_component(download_info):
         verbose=download_info.verbose,
     )
 
-class RustBuild(object):
-    """Provide all the methods required to build Rust"""
-    def __init__(self):
-        self.checksums_sha256 = {}
-        self.stage0_compiler = None
-        self.download_url = ''
+class FakeArgs:
+    """Used for unit tests to avoid updating all call sites"""
+    def __init__(self) -> None:
```

Comment:
Is this syntax supported by python 2.7?

Reply:
whoops, my editor generated this. will fix it